src/utils/tcpros_utils.js

```diff
@@ -181,10 +181,6 @@ let TcprosUtils = {
     else if (!header.hasOwnProperty('md5sum')) {
       return this.serializeString('Connection header missing expected field [md5sum]');
     }
-    // rostopic will send '*' for some commands (hz)
```

Comment:
can you just add a comment here with the TF explanation for why we don't check the type field?

Reply:
Good idea. Done.

Reply:
thanks